app/ceom/photos/views.py

```diff
@@ -73,6 +73,17 @@ def ranges(i):
 def home(request):
     return render(request, 'photos/overview.html')
 
+def leaderboard(request):
+    points = User.objects.annotate(user_points=Count('categoryvote')).order_by("-user_points")
+
+    #Paginator 
+    page_num = request.GET.get('page')
+    paginator = Paginator(points, 100)
+    page = paginator.get_page(page_num)
+
+    return render(request, 'photos/leaderboard.html', {'page': page})    
+#    return render(request, 'photos/leaderboard.html', context=data)
```

Comment:
The commented out line here is actually correct

Start out the function with a blank context dictionary called data, e.g. `data = {}`
Then, you can assign values to it like `data['page_obj'] = paginator.get_page(page_number)`
And finally, the last line becomes `return render(request, 'maps/leaderboard.html', context=data)`
Or `return render(request, 'maps/leaderboard.html', data)`
This isn't strictly necessary for this function in particular, but it's a coding style that I want to keep constant throughout the website

Other than that, looks good to me